umbral-pre-python/docs/index.rst

```diff
@@ -77,14 +113,38 @@ API reference
 
         Verifies the integrity of the fragment using the signing key and, optionally, the delegating and the receiving keys (if they were included in the signature in :py:func:`generate_kfrags`).
 
+    .. py:method:: __bytes__() -> bytes
```

Comment:
I notice GH is drawing a red box around `->`.  Is this some kind of special character?
![Screenshot from 2021-04-21 09-52-13](https://user-images.githubusercontent.com/679404/115591795-5c6f1300-a287-11eb-9199-29394046d2b1.png)

Reply:
I think it's just not used to explicit typehints in the docs. I have to put them in since I have no actual sources, and Sphinx doesn't seem to be able to parse `.pyi`.

Although I'm now thinking that a better approach would be to write a thin wrapper over the Rust bindings. This way:
- docs and typehints will all be in one place and won't need to be repeated
- we can create a more Pythonic API with optional parameters, which are not supported in Rust

Reply:
Added a comment to #30